test/mocks/wrappers/firebase_wrapper_mocks.dart

```diff
@@ -0,0 +1,17 @@
+import 'dart:async';
+
+import 'package:crowdleague/utils/wrappers/firebase_wrapper.dart';
+import 'package:firebase_core/firebase_core.dart';
+
+class FakeFirebaseWrapper implements FirebaseWrapper {
+  final Completer<FirebaseApp> _completer;
+
+  FakeFirebaseWrapper({Completer<FirebaseApp> completer})
+      : _completer = completer ?? Completer<FirebaseApp>();
+
+  /// Returns a [Future] that completes when the supplied [Completer] says so
```

Comment:
you will complete, or i will make you complete, your in my world now future

![198b4daec66737f1010627b9f1a1e0a2_400x400](https://user-images.githubusercontent.com/49229277/94894819-79035380-04cd-11eb-9bab-de86b82b8866.png)

Reply:
Hahahaha with just emoji your review comments are damn funny, the memes are gonna ruin me 🤣